Homework3.10.2024/src/Profesor.java

```diff
@@ -0,0 +1,58 @@
+public class Profesor {
```

Comment:
`Profesor` and `Elev` have a lot of properties in common (firstName, lastName, age etc). You could use inheritance by creating another class with the common properties and extend that class by `Elev` and `Profesor`.